### ISSUE TITLE
Eliminating confusion surrounding presence of scores associated with hardware wallets

### ISSUE THREAD

**Issue by wbnns** [bitcoin-dot-org/bitcoin.org#3136]
There is some [confusion](https://twitter.com/pavlenex/status/1179753704490229761) regarding why hardware wallet scores are presented the way they are. For example, despite the presence of certain characteristics (like in the aforementioned link) why criteria like Privacy or Validation would be rated Neutral.

Rather than have a blanket neutral classification that applies to all hardware wallets for different criteria, we should consider the default software that accompanies each hardware wallet when it's used. In other words, if the default software that accompanies a hardware wallet allows a user to have control over fees when making a transaction with the hardware wallet, the criteria would be rated accordingly. Same with privacy, etc. From a site visitor's perspective, this would be easier to understand than us explaining concepts like hardware signing, etc., and why a privacy rating doesn't apply to the hardware. It would be better to have a "Ok, cool, so I'll be able to have good fee control and decent privacy with this hardware wallet, too" UX instead of having thoughts like "Wait, why is this neutral" or "Why is there no privacy rating like the Android wallets" when reading the page.

In the past it has been discussed whether or not hardware wallets should have their own unique criteria altogether compared to software wallets / if certain criteria should be omitted, however, it's enough of a challenge as it is for users to understand the ratings when they're new to bitcoin. Also, software is required in order to use a hardware wallet anyway. So from the user's perspective, if they purchase a hardware wallet, and the default accompanying software used to make transactions provides them with good privacy and fee control, this should be indicated on the wallet page for that hardware wallet.

Summing up - this issue is proposing to provide a rating for all criteria as described above, rather than a lengthy explanation, the removal/omission of criteria or a neutral score.

Cc: @crwatkins @nvk

**Comment by crwatkins** [comment]
As I have [previously recommended](https://github.com/bitcoin-dot-org/bitcoin.org/issues/3020#issuecomment-503566103) I believe we should eliminate those scores and columns for hardware wallets for exactly the reasons presented in this issue.

While I eventually think we might want to review the "default software", blending that scoring with the hardware will lead to similar confusion issues.

For us to ever recommend the "default software" it would have to be reviewed against our published criteria.  Quite frankly, the reason that this has not been done in the past is that most of it did not meet our listing criteria.  I suspect some, if not most, of that has changed.

However, I continue to recommend the removal of the the neutral columns for hardware wallets.

**Comment by nvk** [comment]
>However, I continue to recommend the removal of the the neutral columns for hardware wallets.
+1

**Comment by nvk** [comment]
Maybe a few other things to consider in regards to grading and extra useful information:
- Required online or not activation
- Air-gapped functionality
- Physical security (ie Secure element or not)
- Compatibility/Wallets (ie PSBT, Electrum, Core, etc...)
- Address formats ( ie 44'|49'|84')
- Known exploits to keep in mind

**Comment by prusnak** [comment]
> Maybe a few other things to consider in regards to grading and extra useful information:

Most of these really depend on the software wallet connected to the hardware wallet and will again lead to confusion Craig is mentioning.

**Comment by nvk** [comment]
>I guess you mean SD card usage by that. From my perspective USB connection is also air-gapped. Both USB and SD card stacks are quite complex, there is no direct advantage of using one over the other. So we might want to have air-gapped via USB, air-gapped via SD card.

USB is by definition connected to a computer, and not air-gapped. Unwired SD/Physical memory transport like that is referred as Sneakernet. But we don't have to argue that here.  We could just inform users how wallets work and let them make their own decisions.

> This depends on the software wallet connected to the hardware wallet and will again lead to confusion Craig is mentioning.

You can't deny this is less than ideal? But again, there is no need to make into a grade. Just show users how each wallets function.

**Comment by crwatkins** [comment]
> Maybe a few other things to consider in regards to grading and extra useful information

In our very recent update to the wallet pages we introduced a binary list of wallet features (Yes or No with no score).   Just to be clear, the original comments in this issue related what we often refer to as our "criteria" or "score" which is non-binary, and has levels of scoring.

Our new feature list seems to be more suited for the items in the list proposed by @nvk than the scoring originally discussed in this issue.  I'm just pointing out this fairly subtle distinction.

**Comment by nvk** [comment]
> Our new feature list seems to be more suited for the items in the list proposed by @nvk than the scoring originally discussed in this issue. I'm just pointing out this fairly subtle distinction.

Thanks @crwatkins, is this recent update in a different branch? I love the idea of binary feature list.

**Comment by crwatkins** [comment]
Nope.  It's live now on bitcoin.org!  The features are fairly well described on the website for users and more explicit details can be [found in the docs](https://github.com/bitcoin-dot-org/bitcoin.org/blob/master/docs/managing-wallets.md) for developers (and are currently mostly software wallet focused).

**Comment by nvk** [comment]
Yes, I saw that doc before. I thought there was something else new I was missing.

**Comment by crwatkins** [comment]
No one has disagreed that the neutral ratings on hardware wallets are confusing .  While in the past I have recommended simply removing the ratings that are not applicable to our currently listed wallets, there could be some hardware wallets (such as the Case Bitcoin Wallet) that these scores would apply.  I recommend that we change the user facing description of "Neutral" to "Not applicable".
